Sources/Smithy/Document.swift

```diff
@@ -6,26 +6,30 @@
 //
 
 import struct Foundation.Data
+import struct Foundation.Date
+import class Foundation.DateFormatter
 import class Foundation.JSONSerialization
 import class Foundation.NSNull
 import class Foundation.NSNumber
 import func CoreFoundation.CFGetTypeID
 import func CoreFoundation.CFBooleanGetTypeID
 
 public enum Document {
```

Comment:
The enum cases here need to cover all of the Smithy types in 2.1-2.12 https://smithy.io/2.0/spec/simple-types.html#simple-types